.github/workflows/ci.yml

```diff
@@ -14,7 +14,8 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-16.04
+    # which version is stable?
+    runs-on: ubuntu-latest
```

Comment:
keep the same with server version ubuntu-20.04 ?

Reply:
18, 20, latest can't work